docs/getting-started/configure-app-with-plugins.md

```diff
@@ -22,62 +22,41 @@ The following steps assume that you have
 [created a Backstage app](./index.md) and want to add an existing plugin
 to it.
 
-We are using the
-[CircleCI](https://github.com/CircleCI-Public/backstage-plugin/tree/main/plugins/circleci)
-plugin in this example, which is designed to show CI/CD pipeline information attached
-to an entity in the software catalog.
+You can find many wonderful plugins out there for Backstage, for example through the [Community Plugins Repository](https://github.com/backstage/community-plugins) and the [Backstage Plugin Directory](https://backstage.io/plugins).
+
+Adding plugins to your Backstage app is generally a simple process, and ideally each plugin will come with its own documentation on how to install and configure it. In this example we will add the [Tech Radar plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/tech-radar/plugins/tech-radar) to our Backstage app.
 
 1. Add the plugin's npm package to the repo:
 
    ```bash title="From your Backstage root directory"
-   yarn --cwd packages/app add @circleci/backstage-plugin
+   yarn --cwd packages/app add @backstage-community/plugin-tech-radar
    ```
 
    Note the plugin is added to the `app` package, rather than the root
    `package.json`. Backstage Apps are set up as monorepos with
-   [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/). Since
-   CircleCI is a frontend UI plugin, it goes in `app` rather than `backend`.
+   [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/). Frontend UI Plugins are generally added to the `app` folder, while Backend Plugins are added to the `backend` folder. In the example above, the plugin is added to the `app` package because we are adding the frontend plugin.
 
-2. Add the `EntityCircleCIContent` extension to the entity pages in the app:
+2. Now, modify your app routes to include the Router component exported from the tech radar, for example:
 
-   ```tsx title="packages/app/src/components/catalog/EntityPage.tsx"
+   ```tsx title="packages/app/src/App.tsx"
    /* highlight-add-start */
-   import {
-     EntityCircleCIContent,
-     isCircleCIAvailable,
-   } from '@circleci/backstage-plugin';
+   import { TechRadarPage } from '@backstage-community/plugin-tech-radar';
    /* highlight-add-end */
 
-   const cicdContent = (
-     <EntitySwitch>
-       {/* ... */}
-       {/* highlight-add-next-line */}
-       <EntitySwitch.Case if={isCircleCIAvailable}>
-         <EntityCircleCIContent />
-       </EntitySwitch.Case>
-       ;{/* highlight-add-end */}
-     </EntitySwitch>
+   const routes = (
+     <FlatRoutes>
+       /* highlight-add-start */
+       <Route path="/tech-radar" element={<TechRadarPage />} />
```

Comment:
```suggestion
       <Route path="/tech-radar" element={<TechRadarPage width={1500} height={800} />} />
```

Reply:
Sorry I did accept it but apparently still something I need to sort out with my sign off email somewhere 😢 thanks for the catch though!